### DISCUSSION TITLE
Preserving naming of clips when copying in Row mode

### DISCUSSION THREAD

**Discussion by nourileiner** [SynthstromAudible/DelugeFirmware#3167] [Ideas]
I love the fact that we can name CV and MIDI clips now.
It would be really nice that the name of a copied clip would stay the same to have a faster workflow. 

Let's say I name a clip DRUM CHORUS 1 and I copy it to the row below to anew section. That would be amazing that the name stays the same to be able to just rename it to DRUM CHORUS 2 for instance. 

Right now you have to retype everything. 

---

That would be also useful if using a template with different MIDI clips assigned to different hardware gear so it's easy to build new sections while keeping the naming consistent.

**Comment by nikodemus** [comment]
Implemented for both row and grid views in https://github.com/SynthstromAudible/DelugeFirmware/pull/3195

Will be in 1.3

**Reply by nourileiner** [reply]
Amazing. Thank you!
